actionpack/lib/action_controller/api.rb

```diff
@@ -0,0 +1,140 @@
+require 'action_view'
+require 'action_controller'
+require 'action_controller/log_subscriber'
+
+module ActionController
+  # API Controller is a lightweight version of <tt>ActionController::Base</tt>,
+  # created for applications that don't require all functionality that a complete
+  # \Rails controller provides, allowing you to create controllers with just the
+  # features that you need for API only applications.
+  #
+  # An API Controller is different from a normal controller in the sense that
+  # by default it doesn't include a number of features that are usually required
+  # by browser access only: layouts and templates rendering, cookies, sessions,
```

Comment:
Yep, this module is added back from jbuilder itself https://github.com/rails/jbuilder/commit/29c0014a9c954c990075d42c45c66075260e924b#diff-524d52316875e7ab48f1778a8ae620cfR14